lib/package.py

```diff
@@ -26,8 +26,6 @@
 
 CONF = config.get_config().CONF
 LOG = logging.getLogger(__name__)
-BUILD_DEPENDENCIES = "build_dependencies"
-DEPENDENCIES = "dependencies"
 
```

Comment:
cool thanks, it's just cause I would like to put more thought on the backwards compatibility matter, but I don't want to block this series either